src/Libraries/Display/Display.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Analysis;
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Interfaces;
+using Autodesk.DesignScript.Runtime;
+using DSCore;
+using Math = DSCore.Math;
+
+namespace Display
+{
+    public class Display :  IGraphicItem
```

Comment:
`Display` has now been moved to its own file. The class has two static constructors, `Display.ByGeometryColor` and `Display.BySurfaceUvsColors`